rules/detekt/src/test/kotlin/io/nlopez/compose/rules/detekt/ComposeModifierNamingCheckTest.kt

```diff
@@ -52,4 +52,39 @@ class ComposeModifierNamingCheckTest {
         val errors = rule.lint(code)
         assertThat(errors).isEmpty()
     }
+
+    @Test
+    fun `errors when a Composable has a single modifier not named modifier but ends with modifier`() {
+        @Language("kotlin")
+        val code =
+            """
+                @Composable
+                fun Something1(myModifier: Modifier) {}
+            """.trimIndent()
+
+        val errors = rule.lint(code)
+        assertThat(errors).hasStartSourceLocation(2, 16)
+
+        assertThat(errors[0]).hasMessage(ComposeModifierNaming.ModifiersAreSupposedToBeCalledModifierWhenAlone)
+    }
+
+    @Test
+    fun `errors when a Composable has a non first modifier named exactly modifier`() {
+        @Language("kotlin")
+        val code =
+            """
+                @Composable
+                fun Something1(myModifier: Modifier, modifier: Modifier) {}
+            """.trimIndent()
+
+        val errors = rule.lint(code)
+        assertThat(errors)
+            .hasStartSourceLocations(
+                SourceLocation(2, 16),
+                SourceLocation(2, 38),
+            )
+
+        assertThat(errors[0]).hasMessage(ComposeModifierNaming.ModifiersAreSupposedToEndInModifierWhenMultiple)
+        assertThat(errors[1]).hasMessage(ComposeModifierNaming.ModifiersAreSupposedToEndInModifierWhenMultiple)
+    }
```

Comment:
I would argue that enforcing this is not the responsibility of this rule, but ComposeParameterOrder, as it's not like the name is wrong but that the parameters are in the wrong order (`modifier` should go before `myModifier`).

I think that rule, ComposeParameterOrder, should already support this case (and if you saw it not being flagged, it's a bug in that rule)

Reply:
Ah yes I agree that this is out of scope for this particular rule. I have removed this test now in commit 1833017
I forgot there was a ComposeParameterOrder rule as I had it disabled.